engine/src/execution_graph/logic_controllers/BatchJoinProcessing.h

```diff
@@ -523,6 +524,18 @@ class JoinPartitionKernel : public kernel {
 						partitioned.push_back(batch_view);
 					}
 				}
+
```

Comment:
I think here you are sending all the partitions to all the workers as you're supposed to. And I think the following block of code is no longer necessary because it seems that it is also sending all the data

Reply:
Yes, that code was actually replicated. I removed it.